app/code/Magento/CatalogUrlRewrite/Plugin/Eav/Model/AttributeSetRepository.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: patrikpihlstrom
+ * Date: 11/10/17
+ * Time: 22:12
+ */
+
+namespace Magento\CatalogUrlRewrite\Plugin\Eav\Model;
```

Comment:
Please fix the namespace. Here is the test result:

```Magento\Test\Integrity\ClassesTest::testClassReferences
Passed: 24399, Failed: 1, Incomplete: 0, Skipped: 0.
Data set: /opt/bamboo/bamboo-agent-home/xml-data/build-dir/MCCE22-AT398-INSE/magento2ce/app/code/Magento/CatalogUrlRewrite/Plugin/Eav/Model/AttributeSetRepository.php
Incorrect namespace usage(s) found in file /opt/bamboo/bamboo-agent-home/xml-data/build-dir/MCCE22-AT398-INSE/magento2ce/app/code/Magento/CatalogUrlRewrite/Plugin/Eav/Model/AttributeSetRepository.php:
CouldNotDeleteException
/opt/bamboo/bamboo-agent-home/xml-data/build-dir/MCCE22-AT398-INSE/magento2ce/dev/tests/static/testsuite/Magento/Test/Integrity/ClassesTest.php:598
/opt/bamboo/bamboo-agent-home/xml-data/build-dir/MCCE22-AT398-INSE/magento2ce/dev/tests/static/testsuite/Magento/Test/Integrity/ClassesTest.php:371
/opt/bamboo/bamboo-agent-home/xml-data/build-dir/MCCE22-AT398-INSE/magento2ce/lib/internal/Magento/Framework/App/Utility/AggregateInvoker.php:56
/opt/bamboo/bamboo-agent-home/xml-data/build-dir/MCCE22-AT398-INSE/magento2ce/dev/tests/static/testsuite/Magento/Test/Integrity/ClassesTest.php:373

/opt/bamboo/bamboo-agent-home/xml-data/build-dir/MCCE22-AT398-INSE/magento2ce/lib/internal/Magento/Framework/App/Utility/AggregateInvoker.php:115
/opt/bamboo/bamboo-agent-home/xml-data/build-dir/MCCE22-AT398-INSE/magento2ce/lib/internal/Magento/Framework/App/Utility/AggregateInvoker.php:70
/opt/bamboo/bamboo-agent-home/xml-data/build-dir/MCCE22-AT398-INSE/magento2ce/dev/tests/static/testsuite/Magento/Test/Integrity/ClassesTest.php:373```

Reply:
I'm not quite sure what the namespace is supposed to be if not `Magento\CatalogUrlRewrite\Plugin\Eav\Model`. I've copied the namespace structure from other plugins, for example `Magento\Catalog\Plugin\Model\ResourceModel`.